frontend/src/component/admin/billing/BillingDashboard/BillingPlan/BillingDetailsEnterpriseConsumption.tsx

```diff
@@ -0,0 +1,40 @@
+import { styled, Typography, Grid, Button } from '@mui/material';
+import { Link } from 'react-router-dom';
+import { GridRow } from 'component/common/GridRow/GridRow';
+import { GridCol } from 'component/common/GridCol/GridCol';
+import LaunchIcon from '@mui/icons-material/Launch';
+import { useInstanceStatus } from 'hooks/api/getters/useInstanceStatus/useInstanceStatus';
+import { formatApiPath } from 'utils/formatPath';
+
+const StyledInfoLabel = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.smallerBody,
+    color: theme.palette.text.secondary,
+}));
+
+const StyledButton = styled(Button)(({ theme }) => ({
+    marginTop: theme.spacing(1),
+    display: 'inline-flex',
+    alignItems: 'center',
+}));
+
+export const BillingDetailsEnterpriseConsumption = () => {
+    const PORTAL_URL = `${formatApiPath('api/admin/invoices')}/portal`;
```

Comment:
I think this can be simplified.

```suggestion
    const PORTAL_URL = formatApiPath('api/admin/invoices/portal');
```

Same as: https://github.com/Unleash/unleash/blob/c6ab2a1cf7740cf22c0be71adb29c8f032082efa/frontend/src/component/admin/invoice/InvoiceList.tsx#L20